include/dos_system.h

```diff
@@ -345,7 +345,7 @@ class DOS_Drive {
 	DOS_Drive();
 	virtual ~DOS_Drive() = default;
 
-	virtual bool FileOpen(DOS_File** file, char* name, uint32_t flags) = 0;
+	virtual bool FileOpen(DOS_File** file, char* name, uint8_t flags) = 0;
```

Comment:
const while you're at it

Reply:
Some of the `FileOpen` functions modify the arguments.